draft-ietf-rtcweb-jsep.xml

```diff
@@ -1044,6 +1084,23 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             be rolled back.
             </t>
 
+            <t>A rollback will disassociate any RtpTransceivers from
```

Comment:
Awkward. Perhaps:
A rollback will disassociate any RtpTransceivers that were associated with m= sections by the application of the rolled-back session description.